backend/app/services/index/Pages2.scala

```diff
@@ -23,6 +23,9 @@ class Pages2(val client: ElasticClient, indexNamePrefix: String)(implicit val ex
       count(textIndexName).query(
         boolQuery().must(
           termQuery(PagesFields.resourceId, uri.value),
+          // Only count documents whose id is of the format `{documentHash}-{pageNumber}`,
+          // to avoid telling the frontend to try and render documents that were uploaded before
+          // the id format changed in https://github.com/guardian/pfi/pull/884 and https://github.com/guardian/pfi/pull/886
           prefixQuery("_id", s"${uri.value}")
```

Comment:
Maddeningly, prefix queries aren't supported on `_id`. I also tried `match` and `query` (with wildcard), but to no avail

Reply:
So I've gone for a new approach: https://github.com/guardian/giant/pull/62/commits/bef96fb89914f02911c3848b3b2a2024790613de